packages/request-node/babel.config.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
+  presets: [['@babel/preset-env', { targets: { node: 'current' } }], '@babel/preset-typescript'],
```

Comment:
`@babel/preset-typescript`
Is this one required? Since we're only passing down `.js` files to Babel.